### PR TITLE
Use environment-badge package.

### PR DESCRIPTION
**Pull request by DFurnes**
### What does this PR do?
This pull request adds the [environment-badge](https://github.com/DoSomething/environment-badge) package I whipped up this morning, which adds a "local", "dev, or "QA" badge to each respective environment to make it easier to tell them apart:

<br/>

Feel free to peek at the [environment-badge](https://github.com/DoSomething/environment-badge) source as well!

### Any background context you want to provide?
I had originally tried this out on our GraphQL explorer at the end of last year, but never rolled it out elsewhere. I remembered it when I was brainstorming opportunity assessments, and decided to just knock it out!

### What are the relevant tickets/cards?
References DoSomething/infrastructure#47.

### Checklist

* [ ] Documentation added for new features/changed endpoints.
* [ ] Added appropriate feature/unit tests.
* [ ] Added screenshot to PR description of related front-end updates on **small** screens.
* [ ] Added screenshot to PR description of related front-end updates on **medium** screens.
* [ ] Added screenshot to PR description of related front-end updates on **large** screens.

### CODE REVIEW

**Comment by DFurnes** [comment]
And updated dependencies in d0fd12f, which shouldn't break anything but worth keeping an eye out!

```
+ contentful-management@5.4.0
+ prettier@1.15.1
+ apollo-client@2.4.5
+ apollo-cache-inmemory@1.3.9
+ date-fns@2.0.0-alpha.25
+ enzyme-adapter-react-16@1.7.0
+ graphql-anywhere@4.1.22
+ enzyme@3.7.0
+ graphql-tag@2.10.0
+ react-dom@16.6.1
+ react-apollo@2.2.4
+ react-redux@5.1.0
+ react@16.6.1
+ react-test-renderer@16.6.1
+ webpack@4.25.1
+ environment-badge@1.0.1
+ pretty-quick@1.8.0
```

File: package.json [hunk 1]
```diff
@@ -68,39 +68,40 @@
...
     "lodash": "^4.17.10",
     "markdown-it": "^8.4.0",
     "markdown-it-footnote": "^3.0.1",
     "markdown-it-for-inline": "^0.1.1",
-    "prettier": "^1.13.7",
+    "prettier": "^1.15.1",
     "prop-types": "^15.6.2",
     "query-string": "^5.1.1",
-    "react": "^16.4.1",
+    "react": "^16.6.1",
```
Comment (mendelB) [inline package.json]: 🎊  (also, just confirming there are no upgrading steps for this minor version, right?)

Comment (DFurnes) [reply package.json]: There are not! Here's the [changelog](https://reactjs.org/blog/2018/10/23/react-v-16-6.html).

**Comment by weerd** [comment]
Very cool!! 💯

**Pull request merged**